src/main/java/org/folio/fqm/service/CrossTenantQueryService.java

```diff
@@ -22,39 +23,61 @@ public class CrossTenantQueryService {
 
   private final SimpleHttpClient ecsClient;
   private final FolioExecutionContext executionContext;
+  private final EntityTypeFlatteningService entityTypeFlatteningService;
+  private final PermissionsService permissionsService;
 
   public List<String> getTenantsToQuery(UUID entityTypeId) {
     String centralTenantId;
     String consortiumId;
+    List<HashMap<String, String>> userTenantHashMaps;
     // Below if-block is necessary to limit cross-tenant querying to instance entity type until we have a way to
     // configure cross-tenant queries by entity type. This can be removed after completion of MODFQMMGR-335.
     if (!entityTypeId.equals(UUID.fromString("6b08439b-4f8e-4468-8046-ea620f5cfb74"))) {
       return List.of(executionContext.getTenantId());
     }
+
+    EntityType entityType = entityTypeFlatteningService.getFlattenedEntityType(entityTypeId, null);
```

Comment:
Nothing actionable here yet, but one of us is going to get a merge conflict 🙂 On my branch, I changed the method signature to take an `EntityType` instead of an ID, so that'll be fun to deal with 😀